test/parallel/test-eslint-snake-case-filenames.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+
+common.skipIfEslintMissing();
+
+const RuleTester = require('../../tools/node_modules/eslint').RuleTester;
+const rule = require('../../tools/eslint-rules/snake-case-filenames');
+
+const message = 'Filename is not in snake_case.';
+const code = 'var foo = "bar"';
+
+new RuleTester().run('snake-case-filenames', rule, {
+  valid: [
+    {
+      code,
+      filename: 'snake-case-filename.js'
+    }
+  ],
+  invalid: [
+    {
+      code,
+      filename: 'camelCaseFilename.js',
+      errors: [{ message }]
```

Comment:
`message` as defined on L10 doesn't include the filename, i.e., it should be `Filename 'camelCaseFilename' is not in snake_case.`.

(But this part of the test doesn't fail anyway because `'camelCaseFilename.js'` fails the `checkIfInTargetDir` check in the rule.)